cmd/kubelet/app/server.go

```diff
@@ -534,7 +548,7 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 				return err
 			}
 			if cloud != nil {
-				klog.V(2).Infof("Successfully initialized cloud provider: %q from the config file: %q\n", s.CloudProvider, s.CloudConfigFile)
+				klog.V(2).InfoS("Successfully initialized cloud provider", "cloud provider", s.CloudProvider, "config file", s.CloudConfigFile)
```

Comment:
Keys should not have spaces. I think the convention is to use camelCase (i.e. `"cloudProvider"`)